cmd/coordinated/metrics.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015-2017 Diffeo, Inc.
+// This software is released under an MIT/X11 open source license.
+
+package main
+
+import (
+	"github.com/diffeo/go-coordinate/coordinate"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var coordinateSummary = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "diffeo",
+		Subsystem: "coordinate",
+		Name:      "coordinate_summary",
```

Comment:
As you probably noticed, this name stutters a lot.  I'd suggest

```go
Namespace: "coordinate",
Name: "work_units",
```

which also matches the Prometheus convention of having a unit as the last part of the name.